netci.groovy

```diff
@@ -78,7 +85,12 @@ Constants.scenarios.each { scenario ->
                 // This call performs test run checks for the CI.
                 Utilities.addXUnitDotNETResults(newJob, '**/testResults.xml')
                 Utilities.addArchival(newJob, "**/testResults.xml")
-                Utilities.setMachineAffinity(newJob, os, Constants.imageVersionMap[os])
+                if(scenario != 'wasm') {
+                    Utilities.setMachineAffinity(newJob, os, Constants.imageVersionMap[os])
+                }
+                else {
+                    Utilities.setMachineAffinity(newJob, 'Windows.10.Wasm.Open', 'latest-or-auto')
```

Comment:
try setMachineAffinity(newJob, 'windows.10.wasm.open') ... I think the overload with 3 args is looking for non-Helix nodes.